pyRMSD/test/testRMSDCalculators.py

```diff
@@ -249,6 +249,32 @@ def test_fitting_symmetry(self):
                                                                              ])
         rmsds = calculator.oneVsFollowing(0)
         numpy.testing.assert_almost_equal(expected_rmsds, rmsds, 12)
+        
+        
+    def test_fitting_symmetry_new(self):
+        protein_skeleton = numpy.array([[0, 0, 0],[0, 1, 0],[0, 0, 1],[1, 0, 0],[1, 1, 1]])
+        thetas = [0, numpy.pi / 6, numpy.pi / 2, numpy.pi]
+        proteins = []
+        # axis is (1, 0, 0)
+        for i in range(4):
+            t = thetas[i]
+            rot_matrix = numpy.array([[1, 0, 0], [0, numpy.cos(t), -numpy.sin(t)], [0, numpy.sin(t), numpy.cos(t)]])
+            proteins.append((rot_matrix @ protein_skeleton.T).T + i * 3)
```

Comment:
Use explicit matrix multiply rather than `@` operator (which doesn't exist in Python 2).

Reply:
Fixed in the new PR